sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedFlux.java

```diff
@@ -312,14 +313,25 @@ private Flux<FeedResponse<T>> byPage(CosmosPagedFluxOptions pagedFluxOptions, Co
                         CosmosDiagnostics diagnostics = feedResponse != null ?
                             feedResponse.getCosmosDiagnostics() : null;
 
+                        FeedResponseDiagnostics feedResponseDiagnostics = ImplementationBridgeHelpers
+                                .CosmosDiagnosticsHelper
+                                .getCosmosDiagnosticsAccessor()
+                                .getFeedResponseDiagnostics(diagnostics);
+
+                        Duration effectiveLatency = Duration.between(startTime.get(), Instant.now()).minus(
```

Comment:
have a question here:

suppose i have 100 items in container, and I am query all of them with page size 10, which means eventually I am going to get 10 pages.  For each response, when I calling the getDuration() API, what is the expect result I should get? Does the duration should only cover for each response, or it is a cumulative of previous response as well? 

`
  container.queryItems(query, JsonNode.class)
      .byPage(10)
      .flatMap(response -> {
           response.getCosmosDiagnostics().getDuration()
      }) 

`

Reply:
It is the time elapsed between when the `Flux` is first subscribed to and when the `Flux` emits a `feedResponse` excluding the time taken to perform a series of `handle` operations. Technically, it is not a cumulation because latency of a later emitted `feedResponse` does not depend feedResponses emitted prior to it.

Reply:
I agree that the later emitted page's duration should not include the previous one. But the testing result does not show the seem, the getDuration() return result keeps increasing. (I think it is because of the startTime start when byPage() is called)
![image](https://user-images.githubusercontent.com/64233642/210459642-a75106ea-3774-4987-9b71-6d304bfd2f66.png)

Reply:
Good point, we should try to isolate the `getDuration()` API for a single feed response. And have separate end to end durations for separate feed responses.

Reply:
Also, calculating end to end latency with subscription and each / success signal might not give us accurate results. Better to calculate these one level down - away from reactor callbacks.